openedx/core/djangoapps/auth_exchange/tests/utils.py

```diff
@@ -2,10 +2,16 @@
 Test utilities for OAuth access token exchange
 """
 
-from social.apps.django_app.default.models import UserSocialAuth
+from django.conf import settings
+from social_django.models import UserSocialAuth, Partial
+
 from third_party_auth.tests.utils import ThirdPartyOAuthTestMixin
 
 
+TPA_FEATURES_KEY = 'ENABLE_THIRD_PARTY_AUTH'
+TPA_FEATURE_ENABLED = TPA_FEATURES_KEY in settings.FEATURES
```

Comment:
I'm honestly not sure - I've seen mixed messages about that. Fine to leave this as-is now though. There's enough other things to work on :)